CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - [\#1958](https://github.com/cosmos/voyager/issues/1958) Kill all processes if one dies to not have dangling processes in development setups @faboweb
 - Fix development warning overlaying the UI too much @faboweb
+- clipboard works in all browsers @jbibla
```

Comment:
I think this suold go in the unreleased section right?

Reply:
yes!